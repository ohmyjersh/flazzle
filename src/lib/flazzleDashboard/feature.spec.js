import React from 'react';
import { shallow } from 'enzyme';
import Feature from './feature';

test('renders correctly', () => {
    const wrapper = shallow(<Feature feature={"chicken"} enabled={true} />);
    expect(wrapper).toMatchSnapshot();
});