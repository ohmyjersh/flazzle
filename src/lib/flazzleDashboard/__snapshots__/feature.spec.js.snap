// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Feature
    enabled={true}
    feature="chicken"
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div>
          <div
                    className="checkbox"
          >
                    <input
                              defaultChecked={true}
                              id="chicken"
                              type="checkbox"
                    />
                    <label
                              htmlFor="chicken"
                    />
          </div>
</div>,
        <span
          className="feature"
>
          Chicken
</span>,
      ],
      "className": "featureContainer",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="checkbox"
>
            <input
                        defaultChecked={true}
                        id="chicken"
                        type="checkbox"
            />
            <label
                        htmlFor="chicken"
            />
</div>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <input
                defaultChecked={true}
                id="chicken"
                type="checkbox"
/>,
              <label
                htmlFor="chicken"
/>,
            ],
            "className": "checkbox",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "defaultChecked": true,
                "id": "chicken",
                "type": "checkbox",
              },
              "ref": null,
              "rendered": null,
              "type": "input",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "htmlFor": "chicken",
              },
              "ref": null,
              "rendered": null,
              "type": "label",
            },
          ],
          "type": "div",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Chicken",
          "className": "feature",
        },
        "ref": null,
        "rendered": "Chicken",
        "type": "span",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div>
            <div
                        className="checkbox"
            >
                        <input
                                    defaultChecked={true}
                                    id="chicken"
                                    type="checkbox"
                        />
                        <label
                                    htmlFor="chicken"
                        />
            </div>
</div>,
          <span
            className="feature"
>
            Chicken
</span>,
        ],
        "className": "featureContainer",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="checkbox"
>
              <input
                            defaultChecked={true}
                            id="chicken"
                            type="checkbox"
              />
              <label
                            htmlFor="chicken"
              />
</div>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <input
                  defaultChecked={true}
                  id="chicken"
                  type="checkbox"
/>,
                <label
                  htmlFor="chicken"
/>,
              ],
              "className": "checkbox",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "defaultChecked": true,
                  "id": "chicken",
                  "type": "checkbox",
                },
                "ref": null,
                "rendered": null,
                "type": "input",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "htmlFor": "chicken",
                },
                "ref": null,
                "rendered": null,
                "type": "label",
              },
            ],
            "type": "div",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Chicken",
            "className": "feature",
          },
          "ref": null,
          "rendered": "Chicken",
          "type": "span",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
